### PR TITLE
Unexport CreateFromIP

### PR DESCRIPTION
**Pull request by ilmanzo**
## Description

fix issue #118

### CODE REVIEW

**Comment by mmlb** [comment]
Hey @ilmanzo thanks for the PR and welcome! I'll take a look shortly :)

**Comment by tstromberg** [comment]
This PR looks great - thank you very much for it ! 

Please revert the unrelated .gitignore and ipxe.efi parts of this change and rebase this change from master to address the merge conflicts. Thank you!

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@
 .idea*/**
 /bin/
 boots
+ipxe/bin/ipxe.efi
```
Comment (nshalman) [inline .gitignore:7]: This is covered by `ipxe/.gitignore` and should be removed here. Looking forward to this landing!

**Comment by ilmanzo** [comment]
sorry but after over 5 months I can't follow this anymore ... closing PR

**Comment by nshalman** [comment]
> sorry but after over 5 months I can't follow this anymore ... closing PR

@ilmanzo understood. Is it okay with you if I clean this up and credit you for the work?

**Comment by ilmanzo** [comment]
Sure, thanks. Looking forward to make more significant contribution to the project!

**Pull request closed without merging**